app/src/main/java/me/ccrama/redditslide/Activities/Wiki.java

```diff
@@ -83,12 +85,26 @@ private void createCustomCss() {
                 .append("color: ".concat(getHexFromColorInt(ta.getColor(2, Color.BLUE))).concat(";"))
                 .append("; }");
         ta.recycle();
-        // TODO: Implement horizontal scrolling to Wiki table views
-        //customCssBuilder.append("table { display: block; overflow-x: auto; white-space: nowrap; }");
+        customCssBuilder.append("table, code { display: block; overflow-x: scroll; }");
+        customCssBuilder.append("table { white-space: nowrap; }");
         customCssBuilder.append("</style>");
         globalCustomCss = customCssBuilder.toString();
     }
 
+    private void createCustomJavaScript() {
+        globalCustomJavaScript = "<script type=\"text/javascript\">" +
+                "window.addEventListener('touchstart', function onSlideUserTouch(e) {" +
+                "var element = e.target;" +
+                "while(element) {" +
+                "if(element.tagName && (element.tagName.toLowerCase() === 'table' || element.tagName.toLowerCase() === 'code')) {" +
```

Comment:
It was pointed out that I call `*.toLowerCase()` twice, extraneously. I should probably move the `tagName` to a variable to avoid that and make the operation marginally faster.